core/src/test/java/com/exadel/aem/toolkit/core/optionprovider/services/impl/EnumOptionSourceResolverTest.java

```diff
@@ -0,0 +1,53 @@
+package com.exadel.aem.toolkit.core.optionprovider.services.impl;
+
+import com.day.cq.commons.jcr.JcrConstants;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ValueMap;
+import org.junit.Rule;
+import org.junit.Test;
+import io.wcm.testing.mock.aem.junit.AemContext;
+
+import com.exadel.aem.toolkit.core.optionprovider.services.impl.enums.TestEnum;
+
+public class EnumOptionSourceResolverTest {
+    @Rule
+    public final AemContext context = new AemContext();
+
+    @Test
+    public void shouldParsePathParameters() {
```

Comment:
Please cover the cases when a user does not provide a value for `texrMember`, `valueMember`, `attributeMember`, or else provides a non-existent method name